src/pages/sighting/Annotations.jsx

```diff
@@ -12,6 +12,7 @@ import AnnotatedPhotograph from '../../components/AnnotatedPhotograph';
 import AnnotationEditor from '../../components/AnnotationEditor';
 import Text from '../../components/Text';
 import ConfirmDelete from '../../components/ConfirmDelete';
+import KickoffAnnotationDialog from './identification/KickoffAnnotationDialog';
```

Comment:
This is actually kicking off ID rather than annotation, right? A little bit of a confusing name.